### ISSUE TITLE
Is it possible to replace addition and subtraction with scalar multiplication?

### ISSUE THREAD

**Issue by amiwrcd** [albertobsd/keysubtracter#5]
Hello Albertobsd! Thank you very much for your projects! Great respect for you!
I am very interested in your ["keysubtracter"](https://github.com/albertobsd/keysubtracter/) is it possible to replace addition and subtraction with scalar multiplication?
Change the code so that the public keys from the file are multiplied by a certain number?

**Comment by albertobsd** [comment]
Are you trying to do point division using the Scalar multiplication with a Inverse multiplicative number right?

That idea is good, I already think in this, and also somebody else request me privately this feature, maybe i will do another proyect like this, i will notify you

**Comment by amiwrcd** [comment]
Yes that's right Albertobsd! I have a large list of compressed public keys and I think with dot division I could get closer to success. I noticed that your projects are working perfectly and fast. I used to try to do point division using Python, but the speed is processing is very low. I think your C ++ code will speed up my work.

**Comment by albertobsd** [comment]
Well scalar multiplication is always 100 times slower than Point addition, even in C/C++ is slow, i can only get some million  of scalar multiplication in somo 10 or 15 minutes, but anyway i will notify you when the other program will be done, please be patient im some busty this days.

Regards!

**Comment by usergoodgithub** [comment]
Hello Albertobsd! Subscribe to the above, good program and thanks! Would like to see a function that multiplies and divides by a scalar. But besides, even with the function of adding and subtracting a key, how can we now customize either only addition or only subtraction?

**Comment by Sadoharu** [comment]
"I would like to see a function that multiplies and divides by a scalar."
I join, I'm also looking forward to it

**Comment by albertobsd** [comment]
> how can we now customize either only addition or only subtraction?

I will add that option
